classes/class.ilTestRunsHelperUIHookGUI.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+class ilTestRunsHelperUIHookGUI extends ilUIHookPluginGUI
+{
+    protected ilTestRunsHelperGUI $gui;
+
+    public function modifyGUI(
+        string $a_comp,
+        string $a_part,
+        array $a_par = []
+    ): void {
+        if ($a_part == 'tabs') {
+            // must be done here because ctrl and tabs are not initialized for all calls
+            global $DIC;
+            if (strtolower($DIC->ctrl()->getCmdClass()) == 'iltestparticipantstablegui') {
```

Comment:
```suggestion
            if (strtolower($DIC->ctrl()->getCmdClass()) === strtolower(ilTestParticipantsTableGUI::class)) {
```
(**::class** Nutzung optional)